### PR TITLE
Fix bad autocorrecting

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: lib/traject/writers/solr_better_json_writer.rb [hunk 1]
```diff
@@ -79,7 +79,7 @@ def send_single(c)
       resp = @http_client.post @solr_update_url, json_package, 'Content-type' => 'application/json'
       # Catch Timeouts and network errors as skipped records, but otherwise
       # allow unexpected errors to propagate up.
-    rescue *skippable_exceptions => e
+    rescue *skippable_exceptions => exception # rubocop:disable Naming/RescuedExceptionsVariableName https://github.com/rubocop/rubocop/issues/11809
```
Comment (jcoyne) [inline lib/traject/writers/solr_better_json_writer.rb:82]: Wasn't this fixed awhile back? Do we still need to disable it?

Comment (cbeer) [reply lib/traject/writers/solr_better_json_writer.rb:82]: 🤷‍♂️ I tried removing them and rubocop didn't like it

Comment (jcoyne) [reply lib/traject/writers/solr_better_json_writer.rb:82]: Ah, I see.

**Review by jcoyne — APPROVED** [review]

File: lib/traject/writers/solr_better_json_writer.rb (hunk 1 above)
Comment (jcoyne) [reply lib/traject/writers/solr_better_json_writer.rb:82]: I created https://github.com/rubocop/rubocop/issues/12042 to argue the point.

**Pull request merged**